pkgs/development/python-modules/tuya-device-sharing-sdk/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, requests
+, paho-mqtt
+, cryptography
+}:
+let
+  pname = "tuya-device-sharing-sdk";
+  version = "0.2.0";
+in
+buildPythonPackage {
+  inherit pname version;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-fu8zh59wlnxtstNbNL8mIm10tiXy22oPbi6oUy5x8c8=";
+  };
+
+  postPatch = ''
+    touch requirements.txt
+  '';
+
+  doCheck = false;
+
+  propagatedBuildInputs = [
+    requests
+    paho-mqtt
+    cryptography
+  ];
```

Comment:
Call this `dependencies` instead and put it below `postPatch` and before `doCheck`.